rotkehlchen/chain/evm/decoding/spark/constants.py

```diff
@@ -0,0 +1,13 @@
+from typing import Final
+
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+
+CPT_SPARK: Final = 'spark'
+
+SPARK_LABEL: Final = 'Spark'
```

Comment:
```suggestion
SPARK_LABEL: Final = CPT_SPARK.capitalize()
```

Reply:
i just used the `SPARK_COUNTERPARTY_DETAILS` instead. Thanks.